### PR TITLE
Add fuctional for running vector

### PR DESCRIPTION
**Pull request by mkryva**

### CODE REVIEW

File: vector.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+      - validator_logs
+    source: |-
+      del(.label)   
+
+sinks:
+  elasticsearch:
+    type: elasticsearch
+    inputs:
+      - remap_field
+    endpoint: http://es-do.observer.cere.network/
```
Comment (alexremn) [inline vector.yaml:20]: @mkryva this endpoint should be available only internally to ingesting nodes.

File: scripts/launch_validator_node.sh [hunk 2]
```diff
@@ -62,6 +62,13 @@ function start_validator_node {
   docker-compose --env-file ./configs/${CONFIG_FILE} up -d add_validation_node_custom
 }
 
+# ============ Start a Vector(log collector) ============
+function start_logging {
```
Comment (AndreiNavoichyk) [inline scripts/launch_validator_node.sh:66]: We don't use this script for now

File: vector.yaml [hunk 3]
```diff
@@ -0,0 +1,22 @@
+sources:
+  validator_logs:
+    type: docker_logs
+    include_containers:
+      - add_validation_node_custom
```
Comment (AndreiNavoichyk) [inline vector.yaml:5]: We have a few containers we are interested in, not only `add_validation_node_custom`. Also `boot_node`, `cere_archive_node`, `cere_full_node `

**Review by AndreiNavoichyk — CHANGES_REQUESTED** [review]
When are you going to migrate and how?

File: scripts/launch_validator_node.sh (hunk 2 above)
Comment (mkryva) [reply scripts/launch_validator_node.sh:66]: kk

**Comment by mkryva** [comment]
It's going to be smooth migrations. We've enabled Elasticsearch support in Grafana, and after some time will disable datadog agent from blockchain nodes.

**Pull request closed without merging**